stable/opsportal/values.yaml

```diff
@@ -114,3 +115,13 @@ kommander:
       traefik.ingress.kubernetes.io/auth-type: forward
       traefik.ingress.kubernetes.io/auth-url: http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/
       traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
+
+  # disable kubefed to keep all items related to kubefed from being created
+  kubefed:
+    enabled: false
+
+###############################################################################
+#  Kubefed: Leave disabled as only kommander should deploy
+###############################################################################
+kubefed:
```

Comment:
I don't think this is needed because you already have it above?

Reply:
had a slack discussion. seems a bit weird that both definitions are needed but.... it is what it is.